cfpb/cfgov-refresh
# Adding margin drop-downs to Call to Action

Adding margin drop-downs to Call to Action


## Changes

- Modified files to add margin drop-downs which allow the user to control margin. 

## Testing

- Add CTA to page
- Select margin options from drop-downs.
- Preview and verify the margin is correct.


## Screenshots

- <img width="1012" alt="screen shot 2017-05-03 at 11 34 46 pm" src="https://cloud.githubusercontent.com/assets/1696212/25689812/539163a4-3059-11e7-84e4-ed4d3799b0f2.png">

## Notes

-  This is largely experimental and the first attempt bring CF control to Wagtail.

## Todos

-

## Checklist
